sdpicedetails.go

```diff
@@ -0,0 +1,15 @@
+// SPDX-FileCopyrightText: 2025 The Pion community <https://pion.ly>
+// SPDX-License-Identifier: MIT
+
+//go:build !js
+// +build !js
+
+package webrtc
+
+type sdpICEDetails struct {
```

Comment:
Can you put this above `extractIceDetails`. Much nicer returning a struct then lots of variables.

The distinct file might confuse users. All of our files right now map to things in `webrtc-pc`